plugins/workspace-plugin/src/ca-cert.ts

```diff
@@ -0,0 +1,53 @@
+/**********************************************************************
+ * Copyright (c) 2019-2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+const PUBLIC_CRT_PATH = '/public-certs';
+const SS_CRT_PATH = '/tmp/che/secret/ca.crt';
+
+const CA_BUNDLE_PATH = '/tmp/ca-bundle.crt';
+
+let bundle: string | undefined = undefined;
+
+export async function getAuthorityCertificate(): Promise<string | undefined> {
+  if (bundle !== undefined) {
```

Comment:
if (bundle) {
 return bundle;
}

?

Reply:
Here are 3 states:
  1 undefined - it needs to check `/tmp/che/secret/ca.crt` and all certificates in `/public-certs`
  2 empty string - no certificates have been found, bundle file hasn't been created
  3 path to the bundle

seems it needs to be reworked a bit to be less confusing